test/rectangles_test.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('chai').assert
+const rectangles = require('../lib/rectangles')
+
+
+describe('functions can return different rectangle properties', function() {
+
+  it('can return the area of a rectangle', function(){
+    var attributes = {width: 10, height: 20};
```

Comment:
you're using `const` on lines 1 and 2, but using `var` here. like we talked about in class today, choose one and stick with it 👍